python/tk_multi_publish2/publish_tree_widget/custom_widget_context.py

```diff
@@ -56,5 +57,22 @@ def set_status(self, status):
         """
         pass
 
-    def set_checkbox_value(self, state):
-        pass
\ No newline at end of file
+    def _on_checkbox_clicked_manually(self):
+        """
+        Callback that fires when the user clicks the checkbox
+        """
+
+        #the stateChanged signal is emitted before clicked() signal, which means
```

Comment:
Nitpick: We always put a space between the `#` and the first letter of our comments.